### PR TITLE
Add send direct message button to profile

### PR DESCRIPTION
**Pull request by melroy89**
Goal: Make the (now working) direct message feature better visible for the end-user, instead of looking in the info menu.

- Add an additional button to the profile, calling it "Direct message" since that covers the meaning much better
  - Button comes with an envelope icon and links to the same `<user>/message` path as the existing send message link.
- Rephrase existing "Send message" in _info.html.twig to "Send direct message" to keep it aligned. 

Example:

image

Example mobile:

image

### CODE REVIEW

File: translations/messages.en.yaml [hunk 1]
```diff
@@ -292,7 +292,7 @@ comment: Comment
 post: Post
 ban_expired: Ban expired
 purge: Purge
-send_message: Send message
+send_message: Send private message
```
Comment (e-five256) [inline translations/messages.en.yaml:295]: I don't believe we should use the word private, as other software doesn't for the reason that it implies the messages are encrypted or secure, instead the word used is "direct message"

Comment (nobodyatroot) [reply translations/messages.en.yaml:295]: agree with @e-five256, there's nothing private about AP at all, lol.

**Comment by e-five256** [comment]
It definitely uses a lot more room on mobile

image

It might be worth making the `<span>` with the text have `<span class="hide-on-mobile">`. That shows the text on desktop but hides it on smaller devices

image

File: translations/messages.en.yaml (hunk 1 above)
Comment (melroy89) [reply translations/messages.en.yaml:295]: I see. Let's call them direct message instead.

Comment (melroy89) [reply translations/messages.en.yaml]: Fixed.

**Comment by melroy89** [comment]
> It might be worth making the `<span>` with the text have `<span class="hide-on-mobile">`.

Agreed. Also done.

**Review by nobodyatroot — APPROVED** [review]

**Pull request merged**